tests/test_plugin.py

```diff
@@ -57,6 +57,20 @@ def test_context_from_instance():
     assert_equals(context, instance.context)
 
 
+def test_families_from_instance():
+    """Instances provide access to their families"""
+
+    context = pyblish.plugin.Context()
+    instance = context.create_instance("MyInstance")
+    instance.data["family"] = "A"
+    families = list(instance.data["family"])
```

Comment:
You're lucky; this would actually make *each character* of the family into individual members of a list. :)

```python
>>> list("abc")
['a', 'b', 'c']
>>>
```

Better make it..

```python
families = [instance.data["family"]]
```

Reply:
Ouch ! Right !!